supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorBackendFactory.scala

```diff
@@ -29,5 +30,19 @@ case class HtCondorBackendFactory(configurationDescriptor: BackendConfigurationD
 
     new SharedFsExpressionFunctions(HtCondorJobExecutionActor.fileSystems, callContext)
   }
+
+  private def resolveCacheProviderProps() = {
+    val cacheEnabled = configurationDescriptor.backendConfig.getBoolean("cache.enabled")
+
+    if (cacheEnabled) {
+      val provider = configurationDescriptor.backendConfig.getString("cache.provider")
+      val cacheFactory = Class.forName(provider)
+        .getConstructor(classOf[com.typesafe.config.Config])
+        .newInstance(configurationDescriptor.backendConfig)
+        .asInstanceOf[CacheActorFactory]
+      Some(cacheFactory.getCacheActorProps())
```

Comment:
I'm really not a fan of using Some/None directly unless one intends to allow for Some(null)